### PR TITLE
fix panic for `select active_branch()` on revision db

### PR DESCRIPTION
**Pull request by JCOR11599**
fix for: https://github.com/dolthub/dolt/issues/5043

it errors now

### CODE REVIEW

File: go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go [hunk 1]
```diff
@@ -786,6 +786,10 @@ func TestDoltRevisionDbScripts(t *testing.T) {
 				Query:    "use mydb/" + commithash,
 				Expected: []sql.Row{},
 			},
+			{
+				Query:          "select active_branch();",
+				ExpectedErrStr: "working set not found",
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go:791]: Not a great error message. This should say "error: no branch selected"

Comment (zachmu) [reply go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go:791]: Tests for this function after `use mydb/mybranch`?

Comment (JCOR11599) [reply go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go]: They are here: https://github.com/dolthub/dolt/blob/main/go/libraries/doltcore/sqle/enginetest/dolt_server_test.go#L122

**Review by zachmu — APPROVED** [review]
LGTM

**Pull request merged**